liblangutil/ErrorReporter.h

```diff
@@ -119,6 +119,9 @@ class ErrorReporter
 		return m_errorCount > 0;
 	}
 
+	// @returns true if error shouldn't be stored
+	bool checkForExcessiveErrors(Error::Type _type);
+
```

Comment:
@chriseth this has been made public so I can use it in the try/catch in`Parser.cpp`. 

For reasons I don't understand, having `paserError()` throw a newly-created `ParserError` is not working, so I am checking for this insde the `catch` as had been done for `hasErrors()`. 

The duplication in the catch part can be driven down by putting it in a macro.  

What do you think?